Jenkinsfile

```diff
@@ -1,4 +1 @@
-#!/usr/bin/env groovy
-
-/* `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library */
-buildPlugin()
+buildPlugin(configurations: buildPlugin.recommendedConfigurations())
```

Comment:
Note that it is not really working in the PR. @basil has no write permissions to the repo

Reply:
Yep, so as-is this is risky and could break master. I'm otp so cannot replay it myself directly.

Reply:
In the worst case I will do the master surgery. There is no pressing need in the release, so I am not that concerned about breaking master

Reply:
Fails in #11 due to `RequireUpperBoundDeps` on `instance-identity`.